### PR TITLE
Devops/create account (completes PRA-42)

### PR DESCRIPTION
**Pull request by AnnaPestova1**
## One Line Description
Created a create account form

## Requirements
Create a registration form (doesn't have to be the final design):
email*, username*, password*, confirm password*

_What this code should do_
Redirect to /home after successful submission

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome
tried to follow this https://www.canva.com/design/DAFpZlWCfIc/trB7WQVj2LnTdN14L0bpRg/edit?amp=&amp=&amp=&utm_source=integration_slack&utm_medium=referral&utm_content=T07EHJ738&utm_campaign=unfurl_document-url_open-in-canva_button&utm_term=c97d7c19-ce97-4496-ab11-bb17fceef54d

_Attach a screenshot here_
not sure that I know how to attach screenshot 
image

## Checklist
- [ ] Create route /newaccount (you can decide what it's name is)
- [ ] Render the create account component that you create at this route
- [ ] Create registration form
- [ ] Redirect to /home after successful submission

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -13,6 +14,7 @@ root.render(
       <Routes>
         <Route path="/" element={<App />} />
         <Route path="/home" element={<Home />} />
+        <Route path="/createAccount" element={<CreateAccount />} />
```
Comment (rebekahcallkacz) [inline src/index.tsx:17]: NON-BLOCKING: Why don't we rename this `/register` since that's what the API endpoint is called also?

**Review by rebekahcallkacz — APPROVED** [review]
I have thoughts on styling, but I am not going to include them now since we are not worrying about styling just yet.

Everything works as expected. My only comment is a renaming suggestion to be consistent with the back end naming conventions!

**Review by rebekahcallkacz — CHANGES_REQUESTED** [review]
Oh, I missed something! Add the Linear issue id to the PR title :)

File: src/index.tsx (hunk 1 above)
Comment (AnnaPestova1) [reply src/index.tsx]: Done!

**Review by rebekahcallkacz — APPROVED** [review]
LGTM

**Review by AnnaSolovykh — APPROVED** [review]
Great job!

**Review by AnnaSolovykh — APPROVED** [review]

File: src/index.tsx (hunk 1 above)
Comment (rebekahcallkacz) [reply src/index.tsx]: 🎉

**Review by rebekahcallkacz — APPROVED** [review]
LGTM

**Pull request merged**